install_msw.sh

```diff
@@ -127,6 +133,28 @@ then
     git config --global core.fileMode false
 fi
 
+if [ "WSL" = "$platform" ]
+then
+    # Install/update packages.
+    packages_list="autoconf automake bsdtar dos2unix doxygen \
+      gdb git libtool make patch pkg-config rsync unzip wget \
+      zip zsh g++-mingw-w64-i686"
+
+    if ! dpkg-query -W $packages_list > /dev/null
```

Comment:
This is wrong, `dpkg-query -W` doesn't check if the package is installed, you can `apt remove autoconf`, for example, and `dpkg-query -W autoconf` will still exit successfully, it only fails for completely unknown packages.

You either need to use something like

     dpkg-query -W -f='${binary:Package}\t${Status}\n' $packages_list | grep -v 'install ok installed'

or just do it unconditionally, which probably shouldn't be a big deal.